### PR TITLE
v0.45.x compatibility

### PR DESCRIPTION
**Pull request by slvrtrn**
Updated tests for 0.45.x compatibility. There were no other breaking changes in the actual driver code.

- Fixed failing tests due to `tu/db-timezone-id` removal
- Fixed enums tests - we have a bit more info there now
- Adjusted the list of green tests in the `deps.edn` blueprint

### CODE REVIEW

**Comment by enqueue** [comment]
@slvrtrn can you rebase, please?

**Comment by slvrtrn** [comment]
Sure. I will do that once we have a stable 0.45.x, as we are using this "include green tests only" tech that needs to be redone in case some new test namespaces are added.
For now, I mostly wanted to verify that the CI will pass, given we do the fixes for an RC.

File: docker-compose.yml [hunk 1]
```diff
@@ -10,3 +10,17 @@ services:
       nofile:
         soft: 262144
         hard: 262144
+<<<<<<< HEAD
```
Comment (mshustov) [inline docker-compose.yml]: oops

**Comment by slvrtrn** [comment]
@enqueue I think we can merge this and then release 0.9.1 that will include #116

**Comment by francisco-mooddie** [comment]
Hi! I don´t know if it is the right place to do this question, but this pull request will fix the fact that on the custom column is not possible do choose the new **datetimeDiff** function?

Thank you!

**Comment by slvrtrn** [comment]
@francisco-mooddie I think we need to implement it on the driver's side. Currently, it is missing.

**Comment by francisco-mooddie** [comment]
@slvrtrn Should I open an Issue or it is something that is already identified and under development? And do you know when it will be released or when it can be developed?

**Comment by slvrtrn** [comment]
@francisco-mooddie yes, please do. If there are any other features from the recent MB versions missing in the driver, please include them in the issue as well. Thanks!

**Comment by djuarezg** [comment]
@slvrtrn is there any other issue or PR to track down the release of 0.9.1? I want to upgrade ASAP and I thought I could build it myself if not otherwise (but README requires the jar to be released by you first)

**Comment by slvrtrn** [comment]
I wanted to include this one as well: #87 
but if it's urgent, I can release it now (including changes from this PR and #116).

**Comment by slvrtrn** [comment]
@djuarezg please try this one: https://github.com/enqueue/metabase-clickhouse-driver/releases/tag/0.9.1

**Comment by djuarezg** [comment]
@slvrtrn I can survive, do not worry and thank you very much! It was more to know when more or less to test and plan the upgrade (I am basically stuck at Metabase 0.45.1 but with driver 0.8.1 as 0.9.0 breaks all dashboards, but 0.8.1 with that newer version breaks certain queries 😔, and not possible to downgrade)

**Comment by slvrtrn** [comment]
@djuarezg please don't hesitate to report any issues you might encounter with 0.9.1. However, it's a bit strange that 0.9.0 did not work with 0.45.x as there were breaking changes related to the tests only...

**Pull request merged**